Task06.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include <cmath>
+
+using std::cout;
+using std::cin;
+using std::endl;
+
+struct Point {
+    int x, y;
+};
+
+void read_point(const Point &p) {

```

Comment:
I suggest  void print(const Point& p) , because read_point may refer to reading the point from the console